java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -11,6 +11,9 @@
  * @see <a href="https://redis.io/commands/?group=set">Set Commands</a>
  */
 public interface SetBaseCommands {
+    /** Redis API keyword used to limit calculation of intersection of sorted sets. */
+    String SET_LIMIT_REDIS_API = "LIMIT";
```

Comment:
I think we call such things this way
```suggestion
    String LIMIT_REDIS_API = "LIMIT";
```

Reply:
SortedSet already has one of these and that causes things to be ambiguous in `BaseClient`, unless we ant to include the class path when we use these